src/Microsoft.Sbom.Api/Config/Args/GenerationArgs.cs

```diff
@@ -59,6 +59,10 @@ public class GenerationArgs : CommonArgs
             "SBOM generation fails.")]
         public string PackageVersion { get; set; }
 
+        [ArgShortcut("ps")]
+        [ArgDescription("Supplier of the package this SBOM represents.")]
```

Comment:
nit: might be clearer to say "Supplier of the package _that_ this SBOM represents."